guides/common/modules/con_smartproxy-overview.adoc

```diff
@@ -0,0 +1,12 @@
+[id="{SmartProxy}-Overview_{context}"]
+= {SmartProxy} overview
+
+With {SmartProxyServers}, you can extend the reach and scalability of your {Project} deployment.
+{SmartProxyServers} provide the following functionalities in a {ProjectName} deployment:
+
+* Mirroring content from {ProjectServer} to establish content sources in various geographical or logical locations.
+By assigning a host system to a {SmartProxyServer}, you can configure this host to receive content and configuration from the {SmartProxy} in their location instead of from the central {ProjectServer}.
```

Comment:
```suggestion
By registering a host to a {SmartProxyServer}, you can configure this host to receive content and configuration from the {SmartProxy} in their location instead of from the central {ProjectServer}.
```